### ISSUE TITLE
Start and stop buggy on older machines.

### ISSUE THREAD

**Issue by Coburn** [gkarsay/parlatype#103]
Using Parlatype from downloaded deb on Ubuntu Jammy, along with the libreoffice extension, using LO 7.3.7.2.

This is not so much a bug report as user feedback.

When I try to run the app on an Ubuntu install that is running from a live USB, there is not enough free memory to use it. I get about two or three sentences typed, and then it loops or freezes.

When I am running it from a desktop computer (currently an older model that may not have the processor speed or RAM that most people are using), a couple of things happen.

1) I typically have to wait 30-45 seconds for Parlatype to begin playing sound after it first loads the file.

2) I have to be very slow, careful, and deliberate when pausing and playing the file. If I click with the mouse or press the spacebar while another action is still executing, Parlatype freezes, and both LibreOffice and Parlatype have to be restarted. While I was getting used to the app, this would happen every few minutes. I can be a nervous clicker, and get two or three clicks ahead of myself, trying to make something happen, or correct a mistake. Other apps I have used (like LO or Rhythmbox) seem to be able to keep up. Parlatype cannot.

To put it another way, if I start or pause Parlatype by clicking or hitting the spacebar in any way that is out of the proper expected order, especially as it is starting up, it freezes and has to be restarted.

This is an elegant app, and it works very well for my purposes. It is also the best app I have found for the Linux universe. However, it costs me money in lost time, having to go back and restart the app every 10 minutes. Would it be possible to make the app more idiot-proof? So that even if I get to clicking like a crazy man, the app remembers all my clicks, and doesn't freeze?

**Comment by Coburn** [comment]
For the record, using the Flatpak version fixed my issues.

**Comment by gkarsay** [comment]
Thanks for your report! This seems to be several issues.

I'd like to answer here on 1) only  and this is very probably an external issue:
https://gitlab.freedesktop.org/pulseaudio/pulseaudio/-/issues/1383

I can reproduce this on a fast machine: If you open a file, wait from that moment 5 seconds and press "Play", it will take approx. 5 seconds until playback starts. The later you press "Play", the longer you wait.

For 2): This is similar to #105, i will continue to address it there.
